### ISSUE TITLE
Option to compress standalone textures?

### ISSUE THREAD

**Issue by mokargas** [KhronosGroup/glTF-Compressor#20]
The tool is fantastic!

However, is it possible to enable the ability to upload textures not attached to a GTLF model and compress them via the tool?

**Comment by ViNeek** [comment]
Hello, I'm glad that you find it useful.

Unfortunately, the tool does not currently support texture compression for standalone images.

A quick workaround is to use the bare minimum GLTF model and add all the textures that you wish to compress. 

I'm including a a minimal GLTF file that you can use to compress multiple textures [quad.zip](https://github.com/KhronosGroup/glTF-Compressor/files/14008293/quad.zip).  It definitely isn't the most elegant solution.

I don't know what image types you require, but for KTX2, the [KTX repo](https://github.com/KhronosGroup/KTX-Software) offers compression tools that operate on standalone textures.

**Comment by mokargas** [comment]
@ViNeek Thanks for the update

I currently use `toktx` CLI to compress my textures to KTX2 as typically I ship my GLBs as geom only (without materials applied due to use case of multiple models and dynamic materials done in code). 

My idea for wanting to do this within the tool is the ability to compare them visually after compression - it would be a little easier and faster than my current workflow to dial in the sweet spot between fidelity and compression. Typically I evaluate a texture's use based on the model's final use in the scene (e.g might have webp, or ktx, or png)

**Comment by MarkCallow** [comment]
> My idea for wanting to do this within the tool is the ability to compare them visually after compression - it would be a little easier and faster than my current workflow to dial in the sweet spot between fidelity and compression. Typically I evaluate a texture's use based on the model's final use in the scene (e.g might have webp, or ktx, or png)

Yes the comparison tool is great. You say you want to evaluate based on the model's final use in the scene. glTF is a way to provide your model to the tool so it can render it and display your textures.

Without the model, just comparing 2 images, I think the comparison tool will be much less useful.
